File: 151004/Eremeichik/rw/src/main/java/com/example/rw/model/dto/sticker/StickerResponseTo.java
package com.example.rw.model.dto.sticker;

import com.example.rw.model.entity.implementations.News;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class StickerResponseTo {

    private String name;
    private Long id;
}

File: 151004/Danilov/src/main/kotlin/com/danilovfa/model/sticker/StickerResponseTo.kt
package com.danilovfa.model.sticker

import kotlinx.serialization.Serializable

@Serializable
data class StickerResponseTo(
    val id: Long,
    val name: String
)

fun Sticker.toResponse() =
    StickerResponseTo(
        id = id,
        name = name
    )
